flask_openapi3/view.py

```diff
@@ -177,12 +177,12 @@ def decorator(func):
 
         return decorator
 
-    def register(self, app: "OpenAPI"):
+    def register(self, app: "OpenAPI", view_class_kwargs: dict = None):
         for rule, (cls, methods) in self.views.items():
             for method in methods:
                 func = getattr(cls, method.lower())
                 header, cookie, path, query, form, body = parse_parameters(func, doc_ui=False)
-                view_func = app.create_view_func(func, header, cookie, path, query, form, body, view_class=cls)
+                view_func = app.create_view_func(func, header, cookie, path, query, form, body, view_class=cls, view_class_kwargs=view_class_kwargs)
```

Comment:
You should notice that every `method` will call `create_view_func`, so we should not use `view_class.as_view`, because every `method` returns a [`view`](https://github.com/pallets/flask/blob/9659b11a45281662c6aa1194dc95b46b1f566ba6/src/flask/app.py#L134), which is unreasonable.

By the way, I also considered using `MethodView` when designing, but it cannot integrate well with OpenAPI.